community.html

```diff
@@ -3,7 +3,7 @@
 layout: default
 comment: >-
   Is your sketch missing from the gallery? Please e-mail
-  <a href="mailto:daniel.wheeler@nist.gov">Daniel Wheeler</a>
+  <a href="mailto:daniel.wheeler@nist.gov">site admin</a>
```

Comment:
Any way to make the `mailto:` address also use `{{ site.admin_email }}`?

Reply:
Done.

Reply:
I meant something more like
```
 <a href="mailto:{{ site.admin_email }}">site admin</a>
```
... basically, replacing your hard-coded address with the variable in every instance instead of only a few.